### DISCUSSION TITLE
manually label left-over duplicates

### DISCUSSION THREAD

**Discussion by j0sien** [asreview/asreview#1297] [Ideas]
While screening large datasets, I manually identify some left-over duplicates in my dataset. I would like as a user to be able to mark these (or even check) manually as 'duplicate', such that the model is not trained twice on this records and that I get a clean set of results.

**Comment by Rensvandeschoot** [comment]
This is being discussed in https://github.com/asreview/asreview/issues/328. Let's continue in that thread
